homeassistant/components/meteoclimatic/const.py

```diff
@@ -51,67 +53,80 @@
     "temp_current": {
         SENSOR_TYPE_NAME: "Temperature",
         SENSOR_TYPE_UNIT: TEMP_CELSIUS,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_TEMPERATURE,
     },
     "temp_max": {
-        SENSOR_TYPE_NAME: "Max Temp.",
+        SENSOR_TYPE_NAME: "Daily Max Temperature",
         SENSOR_TYPE_UNIT: TEMP_CELSIUS,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_TEMPERATURE,
     },
     "temp_min": {
-        SENSOR_TYPE_NAME: "Min Temp.",
+        SENSOR_TYPE_NAME: "Daily Min Temperature",
         SENSOR_TYPE_UNIT: TEMP_CELSIUS,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_TEMPERATURE,
     },
     "humidity_current": {
         SENSOR_TYPE_NAME: "Humidity",
         SENSOR_TYPE_UNIT: PERCENTAGE,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_HUMIDITY,
     },
     "humidity_max": {
-        SENSOR_TYPE_NAME: "Max Humidity",
+        SENSOR_TYPE_NAME: "Daily Max Humidity",
         SENSOR_TYPE_UNIT: PERCENTAGE,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_HUMIDITY,
     },
     "humidity_min": {
-        SENSOR_TYPE_NAME: "Min Humidity",
+        SENSOR_TYPE_NAME: "Daily Min Humidity",
         SENSOR_TYPE_UNIT: PERCENTAGE,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_HUMIDITY,
     },
     "pressure_current": {
         SENSOR_TYPE_NAME: "Pressure",
         SENSOR_TYPE_UNIT: PRESSURE_HPA,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_PRESSURE,
     },
     "pressure_max": {
-        SENSOR_TYPE_NAME: "Max Pressure",
+        SENSOR_TYPE_NAME: "Daily Max Pressure",
         SENSOR_TYPE_UNIT: PRESSURE_HPA,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_PRESSURE,
     },
     "pressure_min": {
-        SENSOR_TYPE_NAME: "Min Pressure",
+        SENSOR_TYPE_NAME: "Daily Min Pressure",
         SENSOR_TYPE_UNIT: PRESSURE_HPA,
+        SENSOR_TYPE_ICON: None,
         SENSOR_TYPE_CLASS: DEVICE_CLASS_PRESSURE,
     },
     "wind_current": {
         SENSOR_TYPE_NAME: "Wind Speed",
         SENSOR_TYPE_UNIT: SPEED_KILOMETERS_PER_HOUR,
         SENSOR_TYPE_ICON: "mdi:weather-windy",
+        SENSOR_TYPE_CLASS: None,
```

Comment:
`SENSOR_TYPES[sensor_type].get(SENSOR_TYPE_CLASS)`

Get will return `None` when it is missing.

```suggestion
```

Reply:
Done.